spec/connectors_async/job_runner_spec.rb

```diff
@@ -173,6 +176,45 @@
         idle_a_bit
       end
     end
+
```

Comment:
I don't see `JobStuckError` in the tests.

Reply:
Good call, lemme add a test for it